bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/handler/HomematicThingHandler.java

```diff
@@ -420,6 +419,37 @@ protected void updateDatapointState(HmDatapoint dp) {
         }
     }
 
+    private Object getValueForConfiguration(HmDatapoint dp) {
```

Comment:
Please add @Nullable to the return type.

Reply:
I've done so (same as in the occasions below), but I guess this binding will need a lot more work before being fully nullability-correct.